posthog/hogql_queries/insights/paths_query_runner.py

```diff
@@ -408,6 +511,28 @@ def get_target_clause(self) -> list[ast.Expr]:
         else:
             return self.get_filtered_path_ordering()
 
+    def get_session_threshold_clause(self) -> ast.Expr:
+        if self.query.pathsFilter.funnelPaths:
+            interval = 14
+            interval_unit = FunnelConversionWindowTimeUnit.day
+
+            if self.query.pathsFilter.funnelActorsQuery.source.funnelsFilter.funnelWindowInterval:
+                interval = self.query.pathsFilter.funnelActorsQuery.source.funnelsFilter.funnelWindowInterval
+                unit = self.query.pathsFilter.funnelActorsQuery.source.funnelsFilter.funnelWindowIntervalUnit
+                interval_unit = funnel_window_interval_unit_to_sql(unit)
+            # TODO: Figure out if funnelWindowDays still used
```

Comment:
Intentionally left-over?

Reply:
I didn't find that we ever send that param, do we?